spec/spec_helper.rb

```diff
@@ -1,5 +1,8 @@
 ENV['RAILS_ENV'] ||= 'test'
 
+require 'coveralls'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.